Server/mods/deathmatch/logic/CGame.h

```diff
@@ -665,7 +665,6 @@ class CGame
 
     int m_iMaxClientTriggeredEventsPerInterval = 100;
     int m_iClientTriggeredEventsIntervalMs = 1000;
-    long long m_lClientTriggeredEventsLastCheck = 0;
 
-    std::map<CPlayer*, int> m_mapClientTriggeredEvents;
+    std::map<CPlayer*, std::pair<int, int>> m_mapClientTriggeredEvents;
```

Comment:
In order to make it more readable I would suggest you to create a new structure instead of using a pair.

Reply:
For example:

```
struct 
{
    long long m_llTicks;
    uint32_t m_uiCounter;
};
```